examples/Parameters/template.yaml

```diff
@@ -0,0 +1,294 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Example project for Powertools for AWS Lambda (.NET) Parameters utility
+
+
+Mappings:
+  Constants:
+    DynamoDB:
+      SingleParamTableName: PowertoolsParametersSingle
+      SingleParamId: my-parameter
+      MultiParamTableName: PowertoolsParametersMultiple
+      MultiParamId: my-hash-key
+    SSM:
+      SingleParamName: /powertools/path/prefix/parameter
+      OtherParamName: /powertools/path/prefix/other-parameter
+      MultiParamPrefix: /powertools/path/prefix
```

Comment:
/powertools-examples/path/prefix

Reply:
renamed it to /powertools-parameters-example/path/prefix